bin/kafka-producer-perf-test.sh

```diff
@@ -17,4 +17,4 @@
 if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ProducerPerformance $@
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance $@
```

Comment:
@omkreddy It's a bit unfortunate that we currently run the tools from within the core project. As @junrao said, we can remove the dependency on `argparse4j`, but that still leaves the `jackson-databind` dependency right? Unless we ban any additional dependencies in the `tools` project, it seems to me like there is no choice, but to copy all dependent jars as well.